src/amo/components/Categories.js

```diff
@@ -78,8 +78,9 @@ export class CategoriesBase extends React.Component {
           className="Categories-list"
           ref={(ref) => { this.categories = ref; }}
         >
-          {categories.map((category, index) => (
-            <li className="Categories-list-item" key={`category-${index}`}>
+          {categories.map((category) => (
+            <li className="Categories-list-item"
```

Comment:
I think if we have newlines in tags at all we should put every attribute on its own line.